dev-docs/bidders/consumable.md

```diff
@@ -1,26 +1,28 @@
 ---
 layout: bidder
 title: Consumable
-description: Prebid Consumable Bidder Adaptor
+description: Prebid Consumable Bidder adapter
 userIds: all
-usp_supported: true
 pbjs: true
 pbs: true
 pbs_app_supported: true
-biddercode: consumable
-tcfeu_supported: true
 schain_supported: true
 coppa_supported: true
+tcfeu_supported: true
+floors_supported: true
+usp_supported: true
 gpp_supported: true
-media_types: banner, video
+biddercode: consumable
+media_types: banner, video, audio
```

Comment:
Please specify

* multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
* safeframes_ok: true/false
* 
if you support multiple formats